File: go/alemedinabjj/footballpoints.go
package main

import (
	"fmt"
)

func main() {
	var wins, draws, losses int
	fmt.Println("Enter the number of wins: ")
	fmt.Scan(&wins)
	fmt.Println("Enter the number of draws: ")
	fmt.Scan(&draws)
	fmt.Println("Enter the number of losses: ")
	fmt.Scan(&losses)
	fmt.Println("The total number of points is: ", wins*3+draws)
}

File: javascript/rafastutz/FootballPoints/footballPoints.js
function footballPoints(wins, draws, losses) {
    const validation = [wins, draws, losses].every(element => typeof element === 'number' && element >= 0) ? "Valid" : "Invalid"
    if(validation === 'Invalid') return "Check your inputs"
    const winPoints = 3
    const drawPoints = 1
    const losePoints = 0

    const totalPoints = winPoints * wins + drawPoints * draws + losePoints * losses
    return totalPoints
}

//Teste
console.log(footballPoints(10,10,10))
console.log(footballPoints(-10,10,10))
console.log(footballPoints(10,10,'abc'))
